sros2/sros2/api/__init__.py

```diff
@@ -44,14 +44,16 @@
 HIDDEN_NODE_PREFIX = '_'
 DOMAIN_ID_ENV = 'ROS_DOMAIN_ID'
 
-NodeName = namedtuple('NodeName', ('node', 'ns', 'fqn'))
-TopicInfo = namedtuple('Topic', ('fqn', 'type'))
+NODE_NAME = namedtuple('NodeName', ('node', 'ns', 'fqn'))
+TOPIC_INFO = namedtuple('Topic', ('fqn', 'type'))
+
+DEFAULT_COMMON_NAME = 'sros2testCA'
```

Comment:
I suggest prepending all of these with underscores so they aren't part of the public API, unless we intend for them to be used-- do we?

Reply:
Yeah currently most (if not all) of the API in this package is public.
It deserves a refactor to split out part into different modules and a full API review of sros2, from the top of my head it's unclear which of these could be useful by other modules for reuse.
So it may be more appropriate to address in a later API overhaul PR spanning the entire repo.

Reply:
Yeah no reason to modify the others until an overhaul PR, but I still think it's a good idea not to add even more public API unless we're sure it should be public. `DEFAULT_COMMON_NAME` probably doesn't need to be, would you agree?

Reply:
Fine by me: https://github.com/ros2/sros2/pull/134/commits/5422fdbbcebe5f834ab0e3b2c4057b01e725db67